### DISCUSSION TITLE
Trigger all recurring transactions/bills on first day of the month

### DISCUSSION THREAD

**Discussion by Stelage** [firefly-iii/firefly-iii#4449] [Firefly III] [answered]
Hi everyone, 

I'v read the documentation, but if I understood correctly, the cron tasks only add the transactions that come in for the current day.

Is it possible to trigger all recurrent transactions/bills on, for example, the first day of the month ? 

The idea behind this is to have an updated monthly chart on the dashboard with the future withdraws automatically added by anticipation.

**Comment by JC5** [comment, marked as answer]
You could run the cron with `--force --date=YYYY-MM-DD` for each day (in the future) that you want to create transactions for.

**Reply by Stelage** [reply]
That's awesome ! Thanks a lot for your fast response.
I'v just tested it and it works just fine !

Before asking my question I was thinking of making a script that would perform the queries on the API but using the already implemented cron is more straightforward.

Should I open a PR aimed to add this feature to the documentation ? :)

**Reply by JC5** [reply]
Sure thing!
